src/Sql/SqlBase.php

```diff
@@ -5,12 +5,15 @@
 use Drupal\Core\Database\Database;
 use Drush\Drush;
 use Drush\Log\LogLevel;
+use Robo\Common\ConfigAwareTrait;
+use Robo\Contract\ConfigAwareInterface;
 use Webmozart\PathUtil\Path;
 
-class SqlBase
+class SqlBase implements ConfigAwareInterface
```

Comment:
SqlBase is neither a commandfile, nor is it created from the DI container. It will therefore be necessary to manually inject the config manager in any code that creates an instance of a class derived from SqlBase.